contracts/HubPool.sol

```diff
@@ -80,43 +85,51 @@ contract HubPool is Testable, Lockable, MultiCaller, Ownable {
     event WhitelistRoute(address originToken, uint256 destinationChainId, address destinationToken);
 
     event InitiateRefundRequested(
-        uint64 indexed relayerRefundId,
         uint64 requestExpirationTimestamp,
-        uint64 poolRebalanceLeafCount,
+        uint64 unclaimedPoolRebalanceLeafCount,
         uint256[] bundleEvaluationBlockNumbers,
         bytes32 poolRebalanceRoot,
         bytes32 destinationDistributionRoot,
         address indexed proposer
     );
     event RelayerRefundExecuted(uint256 relayerRefundId, MerkleLib.PoolRebalance poolRebalance, address caller);
```

Comment:
Can we enumerate the fields in poolRebalance? Also, do you think there are fields we can remove? Do we think these emissions are important for bot verification or anything?